content/en/docs/Creating derivatives/build_iso.md

```diff
@@ -9,62 +9,33 @@ description: >
 
 ![](https://docs.google.com/drawings/d/e/2PACX-1vReZtyNs0imrji-AwnqK0-4ekCKLcKzfnQ_CwiMj93Q7IsycAJHwlNohwCv_hyHnaify7qO-v2Cecg5/pub?w=1223&h=691)
 
-In order to build an iso at the moment of writing, we first rely on [luet-makeiso](https://github.com/mudler/luet-makeiso). It accepts a YAML file denoting the packages to bundle in an ISO and a list of luet repositories where to download the packages from.
+In order to build an iso we rely on [elemental build-iso](https://github.com/rancher-sandbox/elemental) command. It accepts a YAML file denoting the packages to bundle in an ISO and a list of luet repositories where to download the packages from. In addition it can also overlay custom files or use container images from a registry as packages.vim
 
 To build an iso, just run:
 
 ```bash
-docker run -v $PWD:/cOS -v /var/run:/var/run --entrypoint /usr/bin/luet-makeiso -ti --rm quay.io/costoolkit/toolchain ./iso.yaml --image $IMAGE
+docker run --rm -ti -v $(pwd):/build elemental:v0.0.14-36c1b6a --debug build-iso -o /build $IMAGE
 ```
 
-Where `iso.yaml` is the iso specification file, and `--image $IMAGE` is the container image you want to build the ISO for, you might want to check on [how to build bootable images](../creating_bootable_images).
+Where `$IMAGE` is the container image you want to build the ISO for, you might want to check on [how to build bootable images](../creating_bootable_images).
 
-An example of a yaml file using the cos-toolkit opensuse repositories and syslinux:
+`elemental build-iso` command also supports reading a configuration `manifest.yaml` file. It is loaded form the directory specified by `--config-dir` elemental's flag.
 
-```yaml
-packages:
-  rootfs:
-  - system/cos
-  uefi:
-  - live/systemd-boot
-  - live/boot
-  isoimage:
-  - live/syslinux
-  - live/boot
-
-initramfs:
-  kernel_file: "vmlinuz"
-  rootfs_file: "initrd"
-
-image_prefix: "cOS-0."
-image_date: true
-label: "COS_LIVE"
-```
-
-An example using GRUB instead:
+An example of a yaml file using the cos-toolkit opensuse repositories:
```

Comment:
should we include an example file from the elemental repo directly here? We dont have it currently but maybe we should have an examples dir so we could link the most recent examples directly

Reply:
yes that would be nice, so we do not have to worry about updating changes.

Reply:
that reminds me that moving this doc repository inside cOS would help quite a lot, beside code and docs going together, would be helpful also for examples as well